.scripts/travis_setup.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+sudo add-apt-repository -y ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get install -y oracle-java8-installer oracle-java8-set-default
+sudo update-java-alternatives -s java-8-oracle
+export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+
+mysql -u root -e 'create database drupal;'
+mysql -u root -e "GRANT ALL PRIVILEGES ON drupal.* To 'drupal'@'localhost' IDENTIFIED BY 'drupal';"
+
+cd $HOME
+pear channel-discover pear.drush.org
+pear upgrade --force Console_GetoptPlus
+pear upgrade --force pear
+pear channel-discover pear.drush.org
+
+# Drush
+cd /tmp
+php -r "readfile('https://s3.amazonaws.com/files.drush.org/drush.phar');" > drush
+php drush core-status
+chmod +x drush
+sudo mv drush /opt
+sudo ln -s /opt/drush /usr/bin/drush
+
+phpenv rehash
+
+cd $HOME
+drush dl drupal-8.2.2 --drupal-project-rename=drupal
+cd $HOME/drupal
+drush si minimal --db-url=mysql://drupal:drupal@localhost/drupal --yes
+drush runserver --php-cgi=$HOME/.phpenv/shims/php-cgi localhost:8081 &>/tmp/drush_webserver.log &
+
+ln -s $ISLANDORA_DIR modules/islandora
```

Comment:
You need to actually pull down the islandora modules with git here instead of symlinking like we do in the islandora repo's setup.  Line 617-618 in the Travis log show the module can't be found when you try to enable it.